### ISSUE TITLE
cannot reproduce the result

### ISSUE THREAD

**Issue by ZxAndJb** [A-little-star/EchoFree#3]
Dear authors, I am currently reproducing your work and meet several questions, expecting your answer:
1. the SSL loss: in the paper, a pre-trained embedding model is used to compute SSL loss. However, the magnitude of each Intermediate Hidden Layer is different, this leads to a few layers with significantly larger magnitudes dominating the loss, thereby hindering the overall training process.
2. The parameter count reported in the paper cannot be reproduced. I completely follow every word in the paper to reconstruct model, it only counts 158K parameters, compared to the 0.28M in the paper. I understand the trained model weights are not publicly available due to some commercial reasons, but I highly recommend releasing the architecture file if you want this work to be more influential.
